### PR TITLE
eQSL import from multiple QTH nicknames

### PR DESCRIPTION
**Pull request by nolith**
Each eQSL account can define multiple QTH, these are linked accounts.

On top of that, it is also possible to define multiple callsigns using a "/something" suffix. And any of those can still define multiple QTH.

The only common element is the password.

With this MR, when importing from eQSL, each station_callsing/QTH couple will be fetched and imported according to the last QSL received date for that given callsign/QTH.

Screen Shot 2022-08-24 at 18 09 50

### CODE REVIEW

File: application/libraries/Adif_parser.php [hunk 1]
```diff
@@ -111,6 +111,7 @@ public function initialize() //this function locates the <EOH>
 		};
 
         $this->datasplit = preg_split("/<eor>/i", mb_substr($this->data, $this->i, NULL, "UTF-8"));
+		$this->currentarray = 0;
```
Comment (nolith) [inline application/libraries/Adif_parser.php:114]: This is needed to correctly reset the adif parser to handle more than one file

**Comment by nolith** [comment]
Rebased to fix the merge conflict

**Comment by AndreasK79** [comment]
@nolith excellent that you fixed the merge conflict.

@magicbug yolo on this one as well?

**Comment by nolith** [comment]
@AndreasK79 give me some day to update my own installation and test the rebase. I'm not sure if I'll have time today

**Comment by AndreasK79** [comment]
@nolith no worries. Just give us the heads up :)

**Comment by phl0** [comment]
I go for YOLO here as well :)

**Comment by magicbug** [comment]
Cross everything

**Comment by nolith** [comment]
Thanks for merging. Just to close the loop, the rebase was ok. It all works as expected

**Pull request merged**